docs/machine-learning/tutorials/predict-prices-with-model-builder.md

```diff
@@ -0,0 +1,195 @@
+
+1. In the data step of the Model Builder tool, select *File* from the data source dropdown.
+1. Select the button next to the *Select a file* text box and use File Explorer to browse and select the *taxi-fare-test.csv* in the *Data* directory
+1. Choose *fare_amount* in the *Label or Column to Predict* dropdown and navigate to the train step of the Model Builder tool.
+
+## Train the model
+
+The machine learning task used to train the price prediction model in this tutorial is regression. ML.NET supports various regression algorithms. During the model training process, Model Builder will train separate models using different regression algorithms and hyper-parameters to try and train the best performing model for your dataset.
+
+The time required for the model to train is proportionate to the amount of data. Use this chart as guidance to select an adequate value for the `Time to train (seconds)` field:
+
+*Dataset Size  | Dataset Type       | Avg. Time to train*
+------------- | ------------------ | --------------
+0 - 10 Mb     | Numeric and Text   | 10 sec
+10 - 100 Mb   | Numeric and Text   | 10 min
+100 - 500 Mb  | Numeric and Text   | 30 min
+500 - 1 Gb    | Numeric and Text   | 60 min
+1 Gb+         | Numeric and Text   | 3 hour+
+
+1. Because the training data file is more than 10MB, use 600 seconds (10 minutes) as the value for *Time to train (seconds)*.
+2. Select *Start Training*.
+
+Throughout the training process, progress data is displayed in the `Progress` section of the train step.
+
+- Status displays the completion status of the training process.
+- Best accuracy displays the accuracy of the best performing model found by Model Builder so far. Higher accuracy means the model predicted more correctly on test data. 
+- Best algorithm displays the name of the best performing algorithm performed found by Model Builder so far.
+- Last algorithm displays the name of the algorithm most recently used by Model Builder to train the model.
+
+Once training is complete, navigate to the evaluate step.
+
+## Evaluate the model
+
+The result of the training step will be one model which had the best performance. In the evaluate step of the Model Builder tool, the output section, will contain the algorithm used by the best performing model in the *Best Model* entry along with metrics in *Best Model Quality (RSquared)*. Additionally, a summary table containing top five models and their metrics. Visit the following link to learn more about [evaluating model metrics](https://docs.microsoft.com/dotnet/machine-learning/resources/metrics).
+
+If you're not satisfied with your accuracy metrics, some easy ways to try and improve model accuracy are to increase the amount of time to train the model or use more data.
+
+## Use the model for predictions
+
+Two projects will be created in `C:\Users\%USERNAME%\AppData\Local\Temp\MLVSTools` directory.
```

Comment:
Does the user need to know that this is where the projects are? Couldn't this be transparent?

Reply:
They don't. Most likely the user will just add the project to the current solution immediately after training. In the event they don't, they have the option to add the generated projects at their leisure by referencing that directory.

Reply:
#resolved